gsoc/gsoc2016/ideas_juliaastro.md

```diff
@@ -0,0 +1,39 @@
+---
+layout: default
+title:  "Ideas for Julia Astro"
+show_main: false
+ideas_team: JuliaAstro
+---
+
+### Image compression and efficient table reading in FITSIO.jl
+
+*Suggested Mentor(s):* [Kyle Barbary](http://github.com/kbarbary)
```

Comment:
In principle it would be just as backup, so the work load wouldn't be much... I would offer myself... but I have no clue about julia (yet)... ;)